airflow/dags/utils/operators/RmdOperator.py

```diff
@@ -0,0 +1,61 @@
+import os
+import re
+import pandas as pd
+
+from airflow.contrib.operators.ssh_operator import SSHOperator
+from airflow.utils.decorators import apply_defaults
+
+from .operator_utils.job_colors import job_colors
+
+from . import register_build
+
+#############
+## Globals ##
+#############
+
+dags_dir = os.path.join(os.getenv('AIRFLOW_HOME'), "dags")
+
+###############
+## Functions ##
+###############
+
+###############
+## Operators ##
+###############
+
+class RmdOperator(SSHOperator):
+    """
+    The RmdOperator executes the R Markdown file. Note that it is up to the Rmd itself
+    to handle connecting to the database.
+    """
+    @apply_defaults
+    def __init__(self, file_path, **kwargs):
+        self.rmd_file = file_path
+        self.ui_color = job_colors["rmd"]
+        
```

Comment:
Deleting .html output Part 1: add `self.html_output =  file_path.replace('.Rmd', '.html')` to the __init__